tests/tag_api_test.go

```diff
@@ -104,10 +104,10 @@ func (suite *TagApiTestSuite) TestGetTagById() {
 	// Extract the response expectation id
 	expectationId := TestUtilsImpl{}.setupExpectation(request, response, 1)
 
-	tagId := TestUtilsImpl{}.extractIdFrom(request)
+	pathParameters := request.PathParameters
```

Comment:
why do this instead of just `tagId := request.PathParameters["id"][0]`?

Reply:
yeah sure we can do that i thought it would be better to reduce the amount of variables we have in case we have multiple ids.

Reply:
I feel it's better to split off into variables to keep the line doing the request short

Reply:
fixed